content/webapp/services/prismic/transformers/exhibition-guides.ts

```diff
@@ -34,21 +92,21 @@ function transformRelatedExhibition(exhibition): ExhibitionLink {
 export function transformExhibitionGuide(
   document: ExhibitionGuidePrismicDocument
 ): ExhibitionGuide {
+  const genericFields = transformGenericFields(document);
   const { data } = document;
-  // const genericFields = transformGenericFields(document);
 
   const components: ExhibitionGuideComponent[] = data.components?.map(
-    component => {
+    (component, index) => {
       return {
-        number: component.number || [],
+        number: index.toString(), // TODO: Remove once Prismic UI allows us to add number
```

Comment:
Just thinking out loud. Do we need a number field in Prismic or is this (or something like it) sufficient? 

We'd need to take account of the caption stops that don't have numbers associated with them. So we'd need a field in Prismic to determine if a number should be shown or not instead.

Just thinking, what's easier for the content editors.

Reply:
> Just thinking out loud. Do we need a number field in Prismic or is this (or something like it) sufficient?
> 
> We'd need to take account of the caption stops that don't have numbers associated with them. So we'd need a field in Prismic to determine if a number should be shown or not instead.
> 
> Just thinking, what's easier for the content editors.

Good question. I think perhaps replacing the number type with a `select > show/hide number` or a `Boolean` might be a better way to go. In terms of what ends up being better for the content editors, we should probably get folks trying it out on Prismic UI to hear from them what feels easier.